lighthouse-core/report/scripts/perf-x.js

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
hmm. What about this approach:
1. Move perf-x.js to perf-x folder
2. In perf-x/server.js, override `reportGenerator.getReportJS`.
So we can keep perf-x.js in perf-x folder while does not need to import something outside core to report-generator.js